reference/1_application_flow/1_3_synchronization/README.md

```diff
@@ -0,0 +1,140 @@
+- They can also be used for merging & splitting data structures, as [demonstrated later](cranqj/app_flow_sync_split.cranqj)
+
+### Multiplexing & demultiplexing
+
+The ```flow/Multiplexer``` node also combines several inputs into a single output, without synchronization
+- The input is forwarded immediately, retaining the input port the original signal was received on. 
+- It's counterpart is ```flow/Demultiplexer```
+- They both have dynamic input/output ports, allowing the developer to scale them as needed
+
+### Repeater
+
+The ```flow/Repeater``` node duplicates the signal it receives, assigning unique tags to each.
+- It assigns a unique, sequentially indexed tag to each duplicate
+- It also has a derivative for arrays, ```data/array/Repeater```, which repeats the input signal for each array element
+- These nodes are useful for providing external input values for iterators, filters,mappers & reducers
+
+## Example - Synchronizing node inputs
+
+> **_Try out:_**
+>
+> Place node **tutorials/app_flow/syncer_splitter** or **[download .cranqj](cranqj/app_flow_sync_split.cranqj)** 
+
+> **_More info:_**
+>
+> For data manipulation use cases of the syncer/splitter nodes, check out  **[Constructing data - Using the flow/Syncer & Splitter nodes for data manipulation](../../2_constructing_data/2_4_syncer_splitter/README.md)**
+
+
+This example demonstrates the basic usage of the ```flow/Syncer``` & ```flow/Splitter``` nodes.
+
+![](images/2021-07-21-09-05-08.png)
+
+Let's create a node with 2 inputs, and connect them to a ```flow/Syncer```.
+- Set the ```fields``` input to a static value of ```["param1","param2"]``` - the ports ```param1``` & ```param2``` will appear
+- Connect these to the inputs
+- Create an output port called ```synced```, and connect it to the syncer
+
+
+![](images/2021-07-21-09-11-35.png)
+
+We can also place a ```flow/Splitter```, and split up the signal again - it can be configured in the same way as the ```flow/Syncer```.
+
+![](images/2021-07-21-09-16-27.png)
+
+To observe the result, we can wrap this node & drive it with some test data. Note the extra ```flow/Syncer``` here - it takes all 3 arguments produced by the node we just created, to create an output like this:
+
+### Sample output:
+
+```json
+{
+  "param1": "a",
+  "param2": "b",
+  "synced": {
+    "param1": "a",
+    "param2": "b"
+  }
+}
+```
+
+As we can see, the output of the ```flow/Syncer``` is a dictionary of it's input values - as a result, it can also be used for constructing data, as demonstrated in later examples.
+
+## Example - Using parameters with enumeration nodes
+
+> **_Try out:_**
+>
+> Place node **tutorials/app_flow/array_repeater** or **[download .cranqj](cranqj/app_flow_repeater.cranqj)** 
+
+As mentioned in [previously](../1_1_junctions/README.md), iterators & it's derivatives (filters, mappers, reducers) create a new signal with a different tag for each item they process, which cannot be directly synchronized with the original signal. 

```

Comment:
Is the "in" needed in "As mentioned in previously"?

Reply:
Will fix